packages/kit/types/ambient.d.ts

```diff
@@ -14,6 +14,23 @@
  * 	interface Stuff {}
  * }
  * ```
+ * 
+ * Note: Since it's an ambient dts file, you can't use `import` statements. Instead you should use the import() function, for example:
```

Comment:
I think if we want to add this note, it'd make more sense to add it at the end. And you could change the colon in the last sentence to a period then

Reply:
Sorry it took a week! Have done this @benmccann